### PR TITLE
formatted files

### PR DESCRIPTION
**Pull request by eRaMvn**
This PR contains all changes from running pre-commit hooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,26 +51,26 @@
 ![Alt text](docs/images/vulnerable-eb-cnames.png?raw=true "Detect vulnerable ElasticBeanstalk CNAMEs")
 
 ## documentation
-[Manual scans - AWS](manual_scans/aws/README.md)
```
Comment (paulschwarzenberger) [inline README.md:54]: the double spaces at the end of each line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

File: docs/README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 ## documentation
-[Manual scans - AWS](../manual_scans/aws/README.md)
```
Comment (paulschwarzenberger) [inline docs/README.md:2]: the double spaces at the end of each line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

File: docs/integration-tests.md [hunk 3]
```diff
@@ -54,5 +54,5 @@ Then in the test you require the mock you can use the function name (e.g. `moto_
 
 Because we use aws profiles we require the profile name to be set up in boto3 for testing.  A "mocked" aws profile is set up in `integration_tests/conftest.py` in the `aws_credentials` fixture (this is why the other moto fixtures have the `aws_credentials` parameter).  If the code under test needs an aws profile name, please use "mocked".
 
-[back to Automated Tests](automated-tests.md)
```
Comment (paulschwarzenberger) [inline docs/integration-tests.md:57]: the double spaces at the end of this line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

File: docs/unit-tests.md [hunk 4]
```diff
@@ -15,5 +15,5 @@ See [Automated Tests](automated-tests.md) for details on how to set up the tests
 * Where required use the built-in unittest.mock assertions to validate calls - i.e. `mock_object.assert_called_once_with(arg)`
 
 
-[back to Automated Tests](automated-tests.md)
```
Comment (paulschwarzenberger) [inline docs/unit-tests.md:18]: the double spaces at the end of this line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1 +1 @@
-* @ovotech/infosec
\ No newline at end of file
+* @ovotech/infosec
```
Comment (paulschwarzenberger) [inline .github/CODEOWNERS:1]: please update to @domain-protect/admins

**Review by paulschwarzenberger — COMMENTED** [review]
Please see inline comments

File: README.md (hunk 1 above)
Comment (eRaMvn) [reply README.md:54]: so I think this is more of a problem with how markdown was formatted. If we want to have new lines, we should do bullet list

File: docs/README.md (hunk 2 above)
Comment (eRaMvn) [reply docs/README.md]: excluded

File: docs/integration-tests.md (hunk 3 above)
Comment (eRaMvn) [reply docs/integration-tests.md:57]: excluded

File: docs/unit-tests.md (hunk 4 above)
Comment (eRaMvn) [reply docs/unit-tests.md:18]: excluded

File: .github/CODEOWNERS (hunk 5 above)
Comment (eRaMvn) [reply .github/CODEOWNERS]: updated

File: README.md (hunk 1 above)
Comment (paulschwarzenberger) [reply README.md]: My intention was to make it look more like a table of contents, hence not using bullet points. 

However after some research and testing, I discovered that the double spaces can be replaced by `<br>` which forces a line break and still passes the pre-commit check.

I'd suggest using that in the relevant places, which then means there should be no need to exclude these `.MD` / `.md` files from the pre-commit check.

**Review by paulschwarzenberger — APPROVED** [review]
LGTM

**Pull request merged**